src/components/autoSuggest/index.js

```diff
@@ -343,11 +352,7 @@ class AutoSuggest extends React.Component {
             />
             : null
           }
-
-          { this.state.value !== '' && this.state.resultsLength === 0 ?
-            <p className={styles.noResults}>{t('No results found')}</p>
-            : null
-          }
+          {<p className={styles.noResults}>{t(getNoResultMessage())}</p>}
```

Comment:
The translation `t(...)` function should be inside the `getNoResultMessage` on each string. Because of the script that generates the `common.json` file. This way it won't see the strings.

Reply:
Also the `{...}` around the `<p>` are not necessary without the ternary operator.